tests/integration/extensions/test_huggingface_agent.py

```diff
@@ -6,6 +8,7 @@
 
 
 @pytest.mark.integration
+@pytest.mark.xfail(sys.platform.startswith("linux"), reason="problem with 3rd libraries")
```

Comment:
Just to be sure, it's a problem with the tests / VCR, but the hugging face extension will still work on linux right?